microsoft-365/bookings/bookings-in-outlook.md

```diff
@@ -57,21 +57,23 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
 
 1. Bookings with me and Bookings share the same licensing model. However, Bookings doesn't have to be turned on for the organization using tenant settings for users to access Bookings with me. The Bookings app must be enabled for users to have access to Bookings with me.
 
-   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy?view=exchange-ps) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
+   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
 
 2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.
 
    ```PowerShell
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
+    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).
 
    To enable anonymous sharing, use the following command.
 
    ```PowerShell
-     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusyReviewer"}
+     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
```

Comment:
Why not using the default of new teants: "CalendarFreeBusyReviewer"

Reply:
Due to the least privilege principle. If anonymous sharing is disabled, this is the minimum requirement to enable it.
Both issue raisers are complaining about allowing additional permissions to anonymous.

Reply:
Thank you @dariomws  and @ekaplanb